src/log.cpp

```diff
@@ -49,14 +49,14 @@ void Log::fatal( const QString& str )
     self->add( str );
 
     // Write the whole log buffer into a file log.txt
-    QFile file( QStandardPaths::writableLocation( QStandardPaths::TempLocation ) + QDir::separator() + "log.txt" );
+    QFile file( QStandardPaths::writableLocation( QStandardPaths::TempLocation ) + QDir::separator() + "birdtray-log.txt" );
```

Comment:
So we also rename the log file?

Reply:
Yes, this is the new file we create to dump the logs in case of a fatal error. This is not the file that can be configured on the command line. For me, there were many files with `log` in my temp folder, so adding the `birdtray-` prefix helped in finding the log file.